draft-ietf-quic-transport.md

```diff
@@ -1131,6 +1131,13 @@ client MUST terminate with a QUIC_VERSION_NEGOTIATION_MISMATCH error code if
 version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
+The position and size of the version validation fields in the transport
+parameters MUST NOT change between different versions of QUIC.  If a version of
+QUIC needs to define a new format for transport parameters, it MUST define and
+use a new TLS extension rather than redefine the layout of the existing
```

Comment:
I misunderstood what this text was attempting to accomplish, I was thinking of transport parameters as connection ID/packet number/etc.

That being said, I'm not clear on why the transport parameter format can't be tied to QUIC version.  Can you explain more about how this attack works?